settings.gradle

```diff
@@ -48,16 +48,16 @@ include ':instrumentation:akka-http-10.0'
 include ':instrumentation:apache-httpasyncclient-4.0'
 include ':instrumentation:apache-httpclient-3.0'
 include ':instrumentation:apache-httpclient-4.0'
-include ':instrumentation:aws-java-sdk-1.11'
-include ':instrumentation:aws-java-sdk-2.2'
-include ':instrumentation:cdi-1.2'
-include ':instrumentation:classloading'
-include ':instrumentation:classloading:jboss-testing'
-include ':instrumentation:classloading:osgi-testing'
-include ':instrumentation:classloading:tomcat-testing'
+include ':instrumentation:aws-sdk-1.11'
+include ':instrumentation:aws-sdk-2.2'
+include ':instrumentation:cassandra-3.0'
+include ':instrumentation:cdi-testing'
+include ':instrumentation:class-loader'
```

Comment:
`class-loader` or `classloader`... I'm ok with either, but mentally I generally think of it as one word.

Reply:
I agree `class-loader` looks a bit odd by itself, but it is `ClassLoader` and not `Classloader`.

What do you think about `java-class-loader`?

Reply:
also fine with me.